indico/modules/events/registration/models/registrations.py

```diff
@@ -67,6 +67,13 @@ class Registration(db.Model):
                                               ['event_registration.forms.event_id', 'event_registration.forms.id']),
                       {'schema': 'event_registration'})
 
+    @classmethod
+    def get_all_for_event(cls, event):
+        """Retrieve all registrations in all registration forms of an event."""
+        from indico.modules.events.registration.models.forms import RegistrationForm
+        return Registration.find_all(~Registration.is_deleted, ~RegistrationForm.is_deleted,
```

Comment:
`Registration.is_active` (i.e. not rejected/withdrawn/deleted) is probably better than `~Registration.is_deleted`